src/Qowaiv.Validation.TestTools/FluentAssertions/ResultAssertions_T.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Qowaiv.Validation.Abstractions;
+
+namespace FluentAssertions.Qowaiv.Validation
+{
+    /// <summary>Contains a number of methods to assert the (in)validness of a <see cref="Result{TModel}"/>.</summary>
+    public sealed class ResultAssertions<TModel> : ResultAssertionsBase<Result<TModel>>
+    {
+        /// <summary>Creates a new instance of the <see cref="ResultActExtensions"/> class.</summary>
```

Comment:
Is this comment correct?

Reply:
I think so, do you have a better/other idea?